workitem/workitem_repository.go

```diff
@@ -102,6 +117,30 @@ func (r *GormWorkItemRepository) LoadByID(ctx context.Context, id uuid.UUID) (*W
 	return ConvertWorkItemStorageToModel(wiType, res)
 }
 
+// LoadBatchByID returns work items for the given ids
+func (r *GormWorkItemRepository) LoadBatchByID(ctx context.Context, ids []uuid.UUID) ([]*WorkItem, error) {
+	defer goa.MeasureSince([]string{"goa", "db", "workitem", "loadBatchById"}, time.Now())
+	res, err := r.LoadBatchFromDB(ctx, ids)
+	if err != nil {
+		return nil, errs.WithStack(err)
+	}
+	workitems := []*WorkItem{}
+	for _, ele := range res {
+		wiType, err := r.witr.LoadTypeFromDB(ctx, ele.Type)
```

Comment:
Not sure if `LoadTypeFromDB` uses a WIT cache. @pranavgore09 can you double check if it is safe to use it this way? If there's no cache, I'd like to see that we only load each WIT once especially if it is shared by multiple WIs.

Reply:
Yes @kwk it is using cache, please see https://github.com/fabric8-services/fabric8-wit/blob/master/workitem/workitemtype_repository.go#L106